cargo-generate/launch.json

```diff
@@ -0,0 +1,45 @@
+// The format of this file is specified in https://probe.rs/docs/tools/vscode/#start-a-debug-session-with-minimum-configuration
```

Comment:
This URL gives me a 404.

BTW json doesn't allow for comments. Did you verify that it's fine for vscode?